src/main/java/com/agileavengers/icuconnectbackend/model/Post.java

```diff
@@ -0,0 +1,39 @@
+package com.agileavengers.icuconnectbackend.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.sql.Timestamp;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Post {
+    @Id
+    @Column(nullable = false)
+    @GeneratedValue
+    private Long id;
+
+    @ManyToOne
+    private User creator;
+
+    @ManyToOne
+    private Community community;
+
+    private Timestamp creation;
+
+    private String title;
+
+    private String text;
+
```

Comment:
The thumbs up counter is only needed for the dto, as it will be calculated when creating the dto. the same information is stored in the connection between Post and Review. I think it does not make sense to have a counter for thumbsUp in the database, since it will change often and will lead to errors.